lib/tools/adb-emu-commands.js

```diff
@@ -31,6 +34,25 @@ emuMethods.rotate = async function () {
   await this.adbExecEmu(['rotate']);
 };
 
+emuMethods.powerAC = async function (state = 'on') {
+  if (POWER_AC_STATES.indexOf(state) === -1) {
+    log.errorAndThrow(`Wrong power AC state sent ${state}`);
+  }
+  await this.adbExecEmu(['power', 'ac', state]);
+};
+
+emuMethods.powerCapacity = async function (percent = 100) {
+  if (!parseInt(percent) || percent < 0 || percent > 100) {
+    log.errorAndThrow(`Wrong power percent: ${percent}`);
+  }
+  await this.adbExecEmu(['power', 'capacity', parseInt(percent)]);
```

Comment:
I agree. `percent = parseInt(percent, 10)` at the beginning of the function would be better.